docs/modules/mechanics/actions-triggers.mdx

```diff
@@ -493,7 +533,15 @@ trigger an Action.
         <td>
           <label>{`action`}</label>
         </td>
-        <td>Sets an Action.</td>
+        <td>
+          <span
+            className="badge badge--secondary"
+            title="Can be either this attribute or a sub-element."
+          >
+            Property
+          </span>
+          Sets a previously defined action.
```

Comment:
*previously defined* made sense when you could only reference by id, maybe this text can be tweaked
`Sets the action to run when the filter allows`